app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-input.tsx

```diff
@@ -1,14 +1,24 @@
 import { forwardRef } from 'react';
+import Cleave from 'cleave.js/react';
 import { t } from '@18f/identity-i18n';
 
+/**
+ * Maximum length of personal key value, including delimiters.
+ */
+const CODE_LENGTH = 19;
```

Comment:
maybe we could add a function to derive this from `blocks`?

```suggestion
function maxLength(blocks: number[]) {
  const chars = blocks.reduce((acc, v) => acc + v, 0);
  const delims = blocks.length - 1;
  return chars + delims;
}
const BLOCKS = [4, 4, 4, 4];
const CODE_LENGTH = maxLength(BLOCKS);
```

Reply:
That's a good idea! It got me thinking as well: Would Cleave itself offer anything to help with this? [They do](https://github.com/nosir/cleave.js/blob/e3fa6f3637fa6d8ea97ef66ac1957950632b72ab/dist/cleave.js#L1370), and furthermore it seems they would enforce the length anyways, even without an explicit `maxlength` on the input, confirmed with the associated specs. I might just go ahead and remove it altogether.

Reply:
> I might just go ahead and remove it altogether.

Removed in 3f15b44be.